rocAL/rocAL/source/pipeline/image.cpp

```diff
@@ -164,7 +164,7 @@ void Image::update_image_roi(const std::vector<uint32_t> &width, const std::vect
 
         if(height[i] > _info.height_single())
         {
```

Comment:
Follow google coding standards for the if conditions

Reply:
Currently this is the way all if conditions are handled. 
I think this can be issued as a seperate PR